client/index.html

```diff
@@ -214,12 +214,26 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Visual Aids</h2>
```

Comment:
@astorije I honestly like calling it "Visuals" because it is specifically things relating to the visual appearance.